packages/sdk/providers/common.ts

```diff
@@ -3,23 +3,47 @@ import { RequestType } from '@walless/messaging';
 
 import { sendRequest } from '../utils/messaging';
 
+const from = 'walless@sdk';
+
 export const requestConnect = async (options: ConnectOptions) => {
 	return await sendRequest(
 		{
-			from: 'walless@sdk',
+			from,
 			type: RequestType.REQUEST_CONNECT,
 			options,
 		},
 		30000,
 	);
 };
 
-export const requestInstallLayout = async (input: string) => {
+export const requestInstallLayout = async (id: string) => {
 	return await sendRequest(
 		{
-			from: 'walless@sdk',
+			from,
 			type: RequestType.INSTALL_LAYOUT,
-			input,
+			id,
+		},
+		30000,
```

Comment:
make this magic number to a global constant with a meaningful name

Reply:
these timeout maybe varying among different requests, I thought about an enum that define the timeout